_summaries/lang2018pointpillars_2.md

```diff
@@ -1,19 +1,51 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: lang2018pointpillars
 # Please fill out info below
-author: # Your GitHub id
+author: aabayomi
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
-
+<!-- 
 TODO: Summarize the paper:
 * What is the core idea?
 * How is it realized (technically)?
 * How well does the paper perform?
-* What interesting variants are explored?
+* What interesting variants are explored? -->
+
+# Core Idea
+
+The authors proposes as much faster approach by which encoder that learns point cloud representations by stacking learned features as vertical tensors. 
```

Comment:
It will be good to break this line and expand further on each part. Like what is the problem, the paper is trying to solve (3D object detection perfomance), how (using conv 2D layers and using pillars). You can explain a bit the pillar as well as this is kind of novel. What is the novelty in the paper(novel point cloud encoder and network).

Reply:
You can also add that PointPillars can work with any standard 2D convolutional detection architecture.